css/components/buttons.css

```diff
@@ -0,0 +1,33 @@
+.button__primary {
+  background-color: #000;
+  color: #fff;
+  border: none;
+  padding: 0.75rem 2rem;
+  cursor: pointer;
+}
+
+.button__primary:hover {
+  background-color: #F0F0F0;
+  color: #000;
+}
+
+.button__primary:focus {
+  background-color: #F0F0F0;
+  color: #000;
+}
+
+.button__secondary {
+  background-color: #F0F0F0;
+  color: #000;
+  border: none;
+  padding: 0.75rem 2rem;
+  cursor: pointer;
+}
+
+.button__secondary:hover {
+  background-color: #FFF;
+}
+
+.button__secondary:focus {
+  background-color: #FFF;
+}
```

Comment:
you could have done this the same way you did with `:focus, :focus-within`, mind to refactor this properly